ldap/servers/slapd/filterentry.c

```diff
@@ -770,12 +770,6 @@ slapi_vattr_filter_test_ext(
     int rc = 0; /* a no op request succeeds */
     int access_check_done = 0;
 
-    if (only_check_access != 0) {
-        slapi_log_err(SLAPI_LOG_ERR, "slapi_vattr_filter_test_ext",
-            "⚠️  DANGER ⚠️  - only_check_access mode is BROKEN!!! YOU MUST CHECK ACCESS WITH FILTER MATCHING");
```

Comment:
This comment was not incorrect. This is broken, and can lead to access control bypasses.